pom.xml

```diff
@@ -26,6 +27,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <quarkus.version>2.15.3.Final</quarkus.version>
+        <smallrye-config.version>3.1.1</smallrye-config.version>
```

Comment:
This has to align with the SmallRye Config version coming from Quarkus.

Reply:
It seems that the version coming from Quarkus is 2.13.1 so I changed it to that one